app/assets/javascripts/common/reports.js

```diff
@@ -875,14 +875,16 @@ Reports = function (withLtfu) {
       );
       registrationsPeriodEndNode.innerHTML =
         periodInfo.bp_control_registration_date;
-      const breakdown = data.bsBelow200Breakdown[period];
+      const breakdown = data.bsBelow200BreakdownRates[period];
       rbsPPBSPercentNode.innerHTML = this.formatPercentage(
-        breakdown["random"] + breakdown["post_prandial"]
+        breakdown ? breakdown["random"] + breakdown["post_prandial"] : 0
```

Comment:
In case breakdown is undefined (there is no data for the period in the current region), there would be an uncaught exception preventing the rest of the JS from rendering. 

We've added this change so that there are no errors when loading the page, and we can default empty graphs to zeros.